### DISCUSSION TITLE
HtmlText Modules i18n

### DISCUSSION THREAD

**Discussion by rcpacheco** [oqtane/oqtane.framework#3226] [Q&A]
What is the proper way to handle HtmlText modules in a multilanguage applicaton? Shouldn't its table have a LanguageId?

**Comment by sbwalker** [comment]
Oqtane does not support dynamic content localization - it only supports localization for static labels, messages, etc... This is by design - dynamic content localization is not on the roadmap for the framework. This topic was discussed in more detail here: https://github.com/oqtane/oqtane.framework/issues/2499
